### PR TITLE
Introducing SGLang Guru on Gurubase.io

### PR DESCRIPTION
**Pull request by kursataktas**
Hello team,

I'm the maintainer of [Anteon](https://github.com/getanteon/anteon). We have created Gurubase.io with the mission of building a centralized, open-source tool-focused knowledge base. Essentially, each "guru" is equipped with custom knowledge to answer user questions based on collected data related to that tool.

I wanted to update you that I've manually added the [SGLang Guru](https://gurubase.io/g/sglang) to Gurubase. SGLang Guru uses the data from this repo and data from the [docs](https://sglang.readthedocs.io/en/latest/) to answer questions by leveraging the LLM.

In this PR, I showcased the "SGLang Guru", which highlights that SGLang now has an AI assistant available to help users with their questions. Please let me know your thoughts on this contribution.

Additionally, if you want me to disable SGLang Guru in Gurubase, just let me know that's totally fine.

### CODE REVIEW

**Comment by ByronHsu** [comment]
This looks very cool! Would you be interested in adding the guru for https://github.com/linkedin/Liger-Kernel too? I am the maintainer of liger-kernel.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 [![license](https://img.shields.io/github/license/sgl-project/sglang.svg)](https://github.com/sgl-project/sglang/tree/main/LICENSE)
 [![issue resolution](https://img.shields.io/github/issues-closed-raw/sgl-project/sglang)](https://github.com/sgl-project/sglang/issues)
 [![open issues](https://img.shields.io/github/issues-raw/sgl-project/sglang)](https://github.com/sgl-project/sglang/issues)
+[![](https://img.shields.io/badge/Gurubase-Ask%20SGLang%20Guru-006BFF)](https://gurubase.io/g/sglang)
```
Comment (ByronHsu) [inline README.md]: can we change the banner to `ask sglang guru (experimental)` to state that the response might not be 100% correct?

**Comment by kursataktas** [comment]
> This looks very cool! Would you be interested in adding the guru for https://github.com/linkedin/Liger-Kernel too? I am the maintainer of liger-kernel.

Sure, but I can't find detailed documentation for Liger-Kernel. Are there any PDFs, websites, or YouTube videos I can use as data sources for it?

**Comment by ByronHsu** [comment]
Are you on sglang slack https://join.slack.com/t/sgl-fru7574/shared_invite/zt-2ngly9muu-t37XiH87qvD~6rVBTkTEHw? We can chat there

**Comment by kursataktas** [comment]
Hey everyone, have you had a chance to check this out?

**Comment by merrymercy** [comment]
We have moved from readthedocs to https://sgl-project.github.io/. Can you update the docs link in https://gurubase.io/g/sglang? It still points to the old address, which is unavailable now.

**Comment by kursataktas** [comment]
Hey @merrymercy I updated it.

**Comment by merrymercy** [comment]
@kursataktas Thanks for the contribution. It is merged.

**Comment by kursataktas** [comment]
Hi everyone,

I’d like to update you on the release of the Maintainer Panel feature on Gurubase. With this panel, you can add, remove, or update data sources, as well as change the logo and more. You can find the details [here](https://github.com/getanteon/gurubase?tab=readme-ov-file#how-to-claim-a-guru). 

In the near future, I’m planning to include analytics insights in this panel, such as the number of questions asked, the most frequently asked ones, and more. I’ll be sure to update this thread once it’s available. However, in case I miss it, I highly recommend joining our [Discord channel](https://discord.gg/9CMRSQPqx6) to stay updated.

If you’d prefer that I don’t update this thread anymore, please let me know.

**Comment by kursataktas** [comment]
Hello @merrymercy,

We recently released the embeddable "Ask AI" widget, which allows you to integrate your Guru’s Q&A capability into websites, documentation, and more.

The installation process is straightforward, as explained in [this GitHub repository](https://github.com/getanteon/gurubase-widget).

You can see it in action on the [Anteon](https://getanteon.com/) and [NetAlertX](https://netalertx.com/) websites.

**Comment by kursataktas** [comment]
Hello @merrymercy,

I want to share an exciting update about Gurubase. We've recently released several great features that I think you'll find interesting.

### 1. Slack & Discord Integration
You can now add your Guru as a bot to Slack and Discord, allowing you to ask questions instantly. More details here: [Discord Bot Integration](https://docs.gurubase.ai/integrations/discord-bot).

### 2. Analytics Dashboard
Track what your users ask your Guru and gain valuable insights. This feature is available on the "My Gurus" page. If you haven’t maintained your Guru yet, I highly recommend doing so to access your analytics. More information about claiming your Guru can be found [here](https://github.com/Gurubase/gurubase?tab=readme-ov-file#how-to-claim-a-guru).
image

### 3. API Support
All Guru functionalities are now accessible via API. You can ask questions, edit data sources, and more using our API endpoints. Here is the [API Doc](https://docs.gurubase.ai/api-reference/introduction).

### 4. Codebase Indexing
We now support indexing codebases from GitHub repositories to generate answers for related queries. This feature is available for select Gurus. If you don’t see the "Codebase Indexed" label on your Guru edit page and would like access, please let me know, I’d be happy to enable it for you!
image

Let me know if you have any questions or feedback!

**Comment by kursataktas** [comment]
Hey @merrymercy,

I wanted to update you that we’ve released the Gurubase GitHub bot.
This means you can now integrate your Guru into your repository,  so users can ask it questions directly through GitHub issues. By default, the Gurubase bot automatically responds to newly opened issues, but you can customize its behavior if needed.

You can see it in action in [this demo issue](https://github.com/Gurubase/gurubase/issues/248).
More information about the bot can be found [here](https://docs.gurubase.ai/integrations/github-bot).

gurubase-github-bot-demo

-----

_Note: As I mentioned in my first update on this thread, please let me know if you’d prefer not to receive updates here._

**Pull request merged**